app/src/main/java/com/parishod/watomatic/fragment/MainFragment.java

```diff
@@ -182,15 +182,14 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         setNumDays();
 
 
-        if(supportedAppsAdapter != null){
-            getEnabledApps();
-            supportedAppsAdapter.updateList(enabledApps);
+        if(supportedAppsAdapter != null) {
+            supportedAppsAdapter.updateList(getEnabledApps());
```

Comment:
Oh! This is my mistake, this is not required here it should be in onResume only

Reply:
Removed 784a975